packages/common/driver-definitions/src/storage.ts

```diff
@@ -341,6 +346,12 @@ export interface IDocumentServicePolicies {
 	 * Summarizer uploads the protocol tree too when summarizing.
 	 */
 	readonly summarizeProtocolTree?: boolean;
+
+	/**
+	 * Whether the driver supports fetching of snapshot in new format which container all
+	 * contents along with the snapshot tree.
+	 */
+	readonly supportNewSnapshotFormat?: boolean;
```

Comment:
"New' becomes old pretty quickly. Let's give it a name / some identity to refer to in the code.

Reply:
changed the driver policy to this: supportNewGetSnapshotApi